lib/random.gd

```diff
@@ -69,9 +69,86 @@ DeclareCategory( "IsRandomSource", IsComponentObjectRep );
 ##  </ManSection>
 ##  <#/GAPDoc>
 ##
-DeclareOperation( "Random", [IsRandomSource, IsList] );
+DeclareOperation( "Random", [IsRandomSource, IsListOrCollection] );
 DeclareOperation( "Random", [IsRandomSource, IsInt, IsInt] );
 
+##############################################################################
+##
+##  <#GAPDoc Label="InstallMethodWithGlobalRandomGen">
```

Comment:
I don't understand that name. What is a "GlobalRandomGen"? Perhaps this refers to the fact that an alternate method using a global random *source* is installed? Then it should perhaps be `InstallMethodWithGlobalRandomSource` ?
Or perhaps just `InstallMethodWithRandomSource` (indicating it can be used to install any method whose first argument is a random source).

I don't want to bikeshed, but I really don't understand the current name.